diff_drive_controller/src/speed_limiter.cpp

```diff
@@ -126,24 +126,25 @@ namespace diff_drive_controller
         {
           dv_max = v0;
           dv_min = min_deceleration * dt;
-          if (v0 + dv < 0)
+          if (v0 + dv_min < 0)
           {
             dv_min = min_acceleration * (dt - v0 / min_deceleration);
           }
         }
         // Transitioning from negative to positive velocity
-        else if (v0 < 0)
+        else // if (v0 < 0)
```

Comment:
This one is to avoid a warning of `dv_min` and `dv_max` used but not initialized later when we `clamp` to compute `dv`.